pkg/collector/corechecks/networkpath/config.go

```diff
@@ -46,5 +64,23 @@ func NewCheckConfig(rawInstance integration.Data, _ integration.Data) (*CheckCon
 	c.MaxTTL = instance.MaxTTL
 	c.TimeoutMs = instance.TimeoutMs
 
+	c.MinCollectionInterval = firstNonZero(
+		time.Duration(instance.MinCollectionInterval)*time.Second,
+		time.Duration(initConfig.MinCollectionInterval)*time.Second,
+		defaultCheckInterval,
+	)
+	if c.MinCollectionInterval < 0 {
```

Comment:
Should this be `<= 0`?

Reply:
@hmahmood Indeed, you are right, it should be `<= 0`. Fixed here cb3c8c7f99.

That said, it's never `0` when reaching this line since if user provide `0` value configuration, it will fallback to `defaultCheckInterval` value.